packages/perseus/src/util/geometry.ts

```diff
@@ -83,6 +85,31 @@ export function vector(a, b) {
     });
 }
 
+export function sumVectors(...points: Coord[]): Coord {
+    return points.reduce(addVectors, [0, 0]);
+}
+
+export function addVectors(a: Coord, b: Coord): Coord {
+    return [a[0] + b[0], a[1] + b[1]];
+}
+
+export function reverseVector(vector: Coord): Coord {
+    return [-vector[0], -vector[1]];
+}
```

Comment:
We have the `vector` module in `@khanacademy/kmath` that I think would be a better place for these more generic vector functions. In fact, we have `vector.add()` there already (https://github.com/Khan/perseus/blob/0775f5055acf53c9cca865f15dedd5ee5678ce8e/packages/kmath/src/vector.ts#L58). 

Could we move/replace these functions with using `vector`?